### PR TITLE
Docs: Vuepress Integration

### PR DESCRIPTION
**Pull request by hpsing**
Vuepress Integration for the documentation.

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            //   },
+            ],
+          },
+          {
+            title: "Contributing",
+            children: [
+              {
+                title: "Contributing to the docs",
+                url:
+                  "https://github.com/cosmos/cosmos-sdk/blob/master/docs/DOCS_README.md",
```
Comment (noandrea) [inline docs/.vuepress/config.js]:
```suggestion
                  "https://github.com/cosmos/cosmos-sdk/blob/main/docs/DOCS_README.md",
```

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+              },
+              {
+                title: "Source code on GitHub",
+                url: "https://github.com/cosmos/cosmos-sdk/",
```
Comment (noandrea) [inline docs/.vuepress/config.js:134]: why the cosmos sdk for the project url?

File: docs/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "build": "vuepress build"
+  },
+  "keywords": [
+    "elesto",
+    "docs",
+    "documentation",
+    "cosmos-cash"
+  ],
+  "author": "",
+  "license": "ISC",
```
Comment (noandrea) [inline docs/package.json]: What is this license? the vuepress one?

**Review by noandrea — COMMENTED** [review]
Can you add also a github action to build and publish the website on github pages (can be disabled atm)

**Comment by hpsing** [comment]
@noandrea sure, let me add it.

File: docs/.vuepress/config.js (hunk 2 above)
Comment (hpsing) [reply docs/.vuepress/config.js:134]: This is a link in a side nav bar under "Resources" section. Should I remove it?

**Comment by hpsing** [comment]
Will be using mkdocs instead vuepress

**Pull request closed without merging**